src/ol/attribution.js

```diff
@@ -53,23 +55,50 @@ ol.Attribution.prototype.getHTML = function() {
 
 
 /**
+ * When `opt_tileGrid` and `opt_projection` are provided, tile ranges from
+ * tiles that are wrapped around the x-axis will also be matched.
  * @param {Object.<string, ol.TileRange>} tileRanges Tile ranges.
+ * @param {ol.tilegrid.TileGrid=} opt_tileGrid Tile grid.
+ * @param {ol.proj.Projection=} opt_projection Projection.
```

Comment:
The type check has to happen somewhere. I pushed another commit that implements your suggested changes. Let me know which one you like better.